### PR TITLE
feat/slider enhancements

### PR DESCRIPTION
**Pull request by mendyEdri**
## Description
* Added scale animation to Slider's thumb.
* Support Slider-track tap to change value.

### CODE REVIEW

File: src/components/slider/index.js [hunk 1]
```diff
@@ -297,6 +325,37 @@ export default class Slider extends PureBaseComponent {
...
   };
 
+  shouldDoubleSizeByDefault = () => {
+    const {activeThumbStyle, thumbStyle} = this.props;
+    return !activeThumbStyle || !thumbStyle;
+  }
+
+  calculatedThumbActiveScale = () => {
+    const {activeThumbStyle, thumbStyle, disabled} = this.props;
+    if (disabled) {
```
Comment (Inbal-Tish) [inline src/components/slider/index.js:335]: Also if user passed 'disableActiveStyling', no?

File: src/components/slider/index.js [hunk 2]
```diff
@@ -370,14 +438,43 @@ export default class Slider extends PureBaseComponent {
...
+    const {
+      thumbStyle,
+      disabled,
+      thumbTintColor
+    } = this.getThemeProps();
+    return (
+      <Animated.View
+        ref={this.setThumbRef}
+        onLayout={this.onThumbLayout}
+        {...this._panResponder.panHandlers}
```
Comment (Inbal-Tish) [inline src/components/slider/index.js:451]: Moving the panHandlers here means you must touch and drag the thumb itself and not the track. Did you test this on a real device? Is it an easy thing to do? Is the hit slop big enough?

File: src/components/slider/index.js (hunk 1 above)
Comment (mendyEdri) [reply src/components/slider/index.js:335]: Right, fixed

File: src/components/slider/index.js (hunk 2 above)
Comment (mendyEdri) [reply src/components/slider/index.js:454]: Added.
You are right, although it's seems easy to drag it, i've added hitslop for better experience.

Comment (Inbal-Tish) [reply src/components/slider/index.js:454]: One more thing about this. This is a UX change and I think we should pass it by our UX guys. Once they'll approve it we can merge.

Comment (mendyEdri) [reply src/components/slider/index.js:454]: Approved by Yulia 🚀

**Review by Inbal-Tish — APPROVED** [review]

**Pull request merged**